ripd/rip_northbound.c

```diff
@@ -1483,241 +1483,337 @@ const struct frr_yang_module_info frr_ripd_info = {
 	.nodes = {
 		{
 			.xpath = "/frr-ripd:ripd/instance",
-			.cbs.create = ripd_instance_create,
-			.cbs.destroy = ripd_instance_destroy,
-			.cbs.get_next = ripd_instance_get_next,
-			.cbs.get_keys = ripd_instance_get_keys,
-			.cbs.lookup_entry = ripd_instance_lookup_entry,
-			.cbs.cli_show = cli_show_router_rip,
+			.cbs = {
+				.cli_show = cli_show_router_rip,
+				.create = ripd_instance_create,
+				.destroy = ripd_instance_destroy,
+				.get_keys = ripd_instance_get_keys,
+				.get_next = ripd_instance_get_next,
+				.lookup_entry = ripd_instance_lookup_entry,
+			},
```

Comment:
I personally prefer the original style, which is more compact, but I'm not terribly unhappy with this change.

I'm only unsure whether it's worth making unnecessary changes in the code to satisfy legacy compilers, like GCC 4.2 (shipped by some BSDs up to this date due to its license). This is something we can discuss later.